lib/board.rb

```diff
@@ -0,0 +1,56 @@
+class Board
+  attr_accessor :cells
+
+  def initialize
+    reset!
+  end
+
+  def reset!
+    @cells = Array.new(9," ")
+
+  end
+
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+  end
+
+  def position(user_input)
+    cells[user_input.to_i - 1]
+  end
+  
+  def full?
+    cells.all? {|cell| cell != " "}
+
+  end
+
+  def turn_count
+    cells.count {|cell| cell == "X" || cell == "O"}
+
+  end
+
+  def taken?(user_input)
```

Comment:
In such a case, you can avoid specifying `true` or `false` and rather rely on the expression itself which is a boolean:
`position(user_input) == " "` will itself return `true` or `false` based on the validity of that expression